system/src/Grav/Console/Gpm/IndexCommand.php

```diff
@@ -138,7 +150,8 @@ protected function serve(): int
                         'Name' => '<cyan>' . Utils::truncate($package->name, 20, false, ' ', '...') . '</cyan> ',
                         'Slug' => $slug,
                         'Version'=> $this->version($package),
-                        'Installed' => $this->installed($package)
+                        'Installed' => $this->installed($package),
+                        'Enabled' => $this->enabled($package),
```

Comment:
and have something like: `'Enabled' => $this->installed($package) ? $this->enabled($package) : '',` here.

Reply:
@jgonyea If you check here, this change would check all your 3 cases.